wisp-logging/src/main/kotlin/wisp/logging/Logging.kt

```diff
@@ -4,13 +4,18 @@ import mu.KLogger
 import mu.KotlinLogging
 import org.slf4j.MDC
 import org.slf4j.event.Level
+import wisp.sampling.Sampler
 
 typealias Tag = Pair<String, Any?>
 
 inline fun <reified T> getLogger(): KLogger {
     return KotlinLogging.logger(T::class.qualifiedName!!)
 }
 
+fun KLogger.sampled(sampler: Sampler = Sampler.rateLimiting(100L)): KLogger {
+    return SampledLogger(this, sampler)
+}
```

Comment:
This is [implemented similarly to misk](https://github.com/cashapp/misk/blob/master/misk-core/src/main/kotlin/misk/logging/Logging.kt#L18-L20), but I did wonder if it should just be part of `getLogger`, something like:

```kt
inline fun <reified T> getLogger(sampler: Sampler = Sampler.always()): KLogger {
    return SampledLogger(KotlinLogging.logger(T::class.qualifiedName!!), sampler)
}
```